src/unix/freebsd.c

```diff
@@ -274,7 +274,7 @@ int uv__sendmmsg(int fd, struct uv__mmsghdr* mmsg, unsigned int vlen) {
 
 
 int uv__recvmmsg(int fd, struct uv__mmsghdr* mmsg, unsigned int vlen) {
-#if __FreeBSD__ >= 11
+#if __FreeBSD__ >= 11 && !__DragonFly__
```

Comment:
```suggestion
#if __FreeBSD__ >= 11 && !defined(__DragonFly__)
```
(otherwise it'll emit preprocessor warnings on freebsd)